tests/testthat/helper-Reporter.R

```diff
@@ -0,0 +1,62 @@
+test_card1.ReportCard <- function() {
+  card <- ReportCard$new()
+
+  card$append_text("Header 2 text", "header2")
+  card$append_text("A paragraph of default text", "header2")
+  card$append_plot(
+    ggplot2::ggplot(iris, ggplot2::aes(x = Petal.Length, y = Sepal.Length)) +
+      ggplot2::geom_point()
+  )
+  card
+}
+
+test_card2.ReportCard <- local({
+  fun <- function() {
+    card <- ReportCard$new()
+
+    card$append_text("Header 2 text", "header2")
+    card$append_text("A paragraph of default text", "header2")
+    lyt <- rtables::analyze(rtables::split_rows_by(rtables::basic_table(), "Day"), "Ozone", afun = mean)
+    table_res2 <- rtables::build_table(lyt, within(airquality, Day <- factor(Day)))
+    card$append_table(table_res2)
+    card$append_table(iris)
+  }
+  cache <- NULL
+  function() {
+    if (is.null(cache)) cache <<- fun()
+    cache$clone()
+  }
```

Comment:
:information_source: using cache as creating the rtable is slow and would make the test slow.

Alternatively, we could not use this card template for the tests and using some other data type (ggplot, data.table, ...)